db/seed.js

```diff
@@ -35,7 +35,7 @@ class dataGenerator {
   }
 
   generateProductId() {
-    return this.getRandomInt(1, 10000);
+    return this.getRandomInt(1, 100000);

```

Comment:
You might want to keep your product ID within the same range as your teammates (1-100 for example). I'd recommend looking into generating auto-incremented ID's when inserting into your database

Reply:
Will do thanks!